share/spice/nxt_account/nxt_account.js

```diff
@@ -0,0 +1,45 @@
+(function (env) {
+    "use strict";
+
+    var formatNxt = function(amount) {
+        return (amount / 100000000)+" NXT"
+    };
+
+    env.ddg_spice_nxt_account = function(api_result) {
+
+        if (!api_result) {
```

Comment:
Return immediately when "status" is equal to "error" 

![screen shot 2015-04-10 at 8 48 15 am](https://cloud.githubusercontent.com/assets/81969/7087974/6169710e-df5e-11e4-8f66-b44e7f3c76f7.png)